src/GitHub.VisualStudio/Base/TeamExplorerNavigationItemBase.cs

```diff
@@ -28,7 +28,14 @@ public TeamExplorerNavigationItemBase(ISimpleApiClientFactory apiFactory, ITeamE
             IsVisible = false;
             IsEnabled = true;
 
-            OnThemeChanged();
+            try
+            {
+                OnThemeChanged();
+            }
+            catch (ArgumentNullException)
+            {
+                // This throws in the unit test runner.
+            }
```

Comment:
Yeah, in `OnThemeChanged`.
I can imagine a situation where the event is actually triggered during a manual unit test run inside VS (the theme changed event is sometimes triggered automatically when VS syncs settings in ze cloud), so we might end up blowing up in the handler that's right after this line, as well.